magprime/automated_emails.py

```diff
@@ -51,3 +51,6 @@ def __init__(self, event):
 
 AutomatedEmail(Attendee, 'Want to help run MAGFest poker tournaments?', 'poker.txt',
                lambda a: a.has_shifts_in(c.TABLETOP), sender='tabletop@magfest.org')
+
+StopsEmail('MAGFest Staff Support', 'staff_support.txt',
+           lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted)
```

Comment:
Do we want to send this out to ALL non-trusted Staff Support people, or only people for whom this is their first year in Staff Support?  We could change this to look at ``a.past_years_json`` which would tell us whether or not they took any shifts there in the past.

Not a huge deal if longtime Staff Support people get this email, but it would probably be cleaner to only send this to new folks.